GSoC-2025/brainglobe: Update atlas packaging scripts to atlas API v2 (Swapnaneel Patra).md

```diff
@@ -0,0 +1,99 @@
+
+## brainglobe: Update atlas packaging scripts to atlas API v2 (Swapnaneel Patra)
+
+## Personal details
+
+- **Full name**: Swapnaneel Patra
+- **Email**: <swapnaneel06@gmail.com>
+- **GitHub username**: thisisrick25
+- **Zulip username**: Swapnaneel
+- **Location & time-zone**: Kolkata, India, GMT+5:30
+- **Personal website / project portfolio**: [github.com](https://github.com/thisisrick25)
+- **Code contribution**: [https://github.com/brainglobe/brainglobe-atlasapi/pull/555](https://github.com/brainglobe/brainglobe-atlasapi/pull/555)
+- **Proposal discussion link**: https://github.com/neuroinformatics-unit/gsoc/pull/56
+
+## Project proposal
+
+_Length: max 1 page_
+
+- **Synopsis**
+
+This project aims to update the BrainGlobe Atlas API by integrating the OpenMINDS SANDS standard for neuroanatomical atlases.
+Currently, atlas packaging scripts convert public atlas data to a limited standard format; by adapting these scripts to produce output compliant with OpenMINDS SANDS,
+The project will enable consistent, interoperable data representations across BrainGlobe tools.
+This standardization is critical for improving data accessibility and integration within the neuroinformatics community.
+The deliverables include modifications to the Atlas packaging Python code, adaptation of at least one packaging script to use the new functionality,
+comprehensive tests, and updated documentation.
+The open source community will benefit through improved data interoperability, ease of integration for new atlases,
+and enhanced reproducibility in neuroanatomical research.
+
+- **Implementation timeline**
+
+    1. A bullet point list with **minimal set of deliverables**
+        - Update atlas packaging Python code to output OpenMINDS SANDS compliant files.
+        - Adapt at least one existing packaging script to utilize the new functionality.
+        - Develop and run unit and integration tests covering the updated functionality.
+        - Update and expand documentation to reflect new usage and features.
+
+    2. Additional **stretch goals** or "if time allows" deliverables
+        - Extend support to additional packaging scripts within the BrainGlobe ecosystem.
+        - Develop conversion utilities for legacy atlas data.
```

Comment:
We wouldn't need this if every packaging script was updated. All existing atlases are created using one of the existing scripts.